docs/index.md

```diff
@@ -121,6 +121,9 @@ For a more comprehensive list of open-source projects using *pydantic* see the
 
 Podcasts and videos discussing pydantic.
 
+[Talk Python To Me](https://talkpython.fm/episodes/show/313/automate-your-data-exchange-with-pydantic){target=_blank}
+: Micheal Kennedy and Samuel Colvin, the creator of *pydantic*, dive into the history of pydantic and its many uses and benefits.
```

Comment:
Typo in Michael's name in both the doc and the change file

Reply:
Fixed typo with commit fa2428344f52c65271fd0444032f40f1c81f854f